test/Serilog.Tests/Core/LoggerTests.cs

```diff
@@ -317,6 +318,54 @@ public void NullMessageTemplateParametersDoNotBreakBinding()
         // ReSharper restore StructuredMessageTemplateProblem
     }
 
+    // https://github.com/serilog/serilog/issues/2019
+    [Fact]
+    public void Two_Dimensional_Array_Should_Be_Logger_As_Sequence()
+    {
+        var evt = DelegatingSink.GetLogEvent(l =>
+        {
+            var a = new object[3, 2] { { "a", "b" }, { "c", "d" }, { "e", "f" } };
+            l.Error("{@Value}", a);
+        });
+
+        evt.Properties.Count.ShouldBe(1);
+        var arr = evt.Properties["Value"].ShouldBeOfType<SequenceValue>();
+        arr.Elements.Count.ShouldBe(6);
+        arr.LiteralValue().ShouldBe("[a,b,c,d,e,f]");
```

Comment:
Would expect `[[a,b],[c,d],[e,f]]` here I think?

Reply:
I did it intentionally since any array is just a sequence in terms of serilog regardless dimensions. Nevertheless I can change implementation to emit sequence of sequences if you like.

Reply:
done